scripts/gef.sh

```diff
@@ -22,13 +21,19 @@ if [ -f "${HOME}/.gdbinit" ]; then
 fi
 
 if [ $wget_found -eq 1 ]; then
+    latest_tag=$(wget -q -O- "https://api.github.com/repos/hugsy/gef/tags" | grep "name" | head -1 | sed -e 's/"name": "\([^"]*\)",/\1/' -e 's/\s*//')
+
     # Get the hash of the commit
+    branch="${latest_tag}"
```

Comment:
why not just use `latest_tag` in the download step?

Reply:
This is a first quick step, next one would be to re-use the saame `getopt` thingy like in the gef-extra script to allow downloading other tag from cli.